Modelplace-Cloud-API-Python/README.md

```diff
@@ -0,0 +1,65 @@
+# Modelplace Cloud API
+​
+This page contains a Python Application that uses Modelplace.AI Cloud API.  
+
+Modelplace.AI Cloud API is a simple HTTP interface with an API key and an image or video as input and JSON with results and visualization as output. Explore [documentation] and this Python example to integrate AI models into your application or workflow.  
+
```

Comment:
Please, record your terminal window of how you run the code, convert it to GIF and place here. 

Please, compress your GIF before publishing to avoid large files. 

*Maybe,* webp can be used here